scripts/jenkins/motr-hare-1node-cluster-rocky-linux.groovy

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env groovy
+pipeline { 
+    agent {
+        node {
+           label 'ssc-vm-g3-rhev4-1327'
+        }
+    }
+	
+    parameters {
+        string(name: 'NODE_HOST', defaultValue: '', description: 'Node 1 Host FQDN',  trim: true)
+        string(name: 'NODE_USER', defaultValue: '', description: 'Host machine root user',  trim: true)
+        string(name: 'NODE_PASS', defaultValue: '', description: 'Host machine root user password',  trim: true)
+    }
+
+    options {
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        ansiColor('xterm') 
+        buildDiscarder(logRotator(numToKeepStr: "30"))
+    }
+
+    environment {
+        NODE_PASS = "${NODE_PASS.isEmpty() ? NODE_DEFAULT_SSH_CRED_PSW : NODE_PASS}"
+    }
+
+    stages {
+        stage('Checkout') {
+            steps {
+                checkout([$class: 'GitSCM', branches: [[name: "EOS-28455"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'PathRestriction', excludedRegions: '', includedRegions: 'scripts/third-party-rpm/.*']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: "https://github.com/rkothiya/cortx-motr"]]])
+            }
+        }
+        stage ('Build rpm packages') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'to build motr and hare rpm', script: '''
+                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} hostname
+                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} git clone https://github.com/rkothiya/cortx-motr
+                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} "cd /root/cortx-motr ;  /root/cortx-motr/scripts/build-prep-1node.sh -dev"
+                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} "hctl bootstrap --mkfs singlenode.yaml"
+                '''
```

Comment:
Can you add the steps of IO for this test ?

Reply:
# dd if=/dev/urandom of=/tmp/128M bs=1M count=128
128+0 records in
128+0 records out
134217728 bytes (134 MB, 128 MiB) copied, 0.958854 s, 140 MB/s

# /opt/seagate/cortx/hare/libexec/m0crate-io-conf > crate.yaml
# ./cortx-motr/utils/m0crate -S crate.yaml